src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -373,7 +373,7 @@ private function validateAndDenormalize(string $currentClass, string $attribute,
                 return null;
             }
 
-            $collectionValueType = $type->isCollection() ? $type->getCollectionValueType() : null;
+            $collectionValueType = $collectionValueType = $type->isCollection() ? $type->getCollectionValueTypes()[0] ?? null : null;
```

Comment:
This line is suspect to me: double ` $collectionValueType = $collectionValueType = `

Reply:
Thanks :+1: